tests/unit/mixins/component-query-manager-test.js

```diff
@@ -9,7 +9,7 @@ moduleFor(
   'Unit | Mixin | component query manager', {
     needs: ['service:apollo'],
     beforeEach() {
-      // needed to set up config since initializers don't run here
+      // needed to set up config since environment won't be initialized in this type of test
```

Comment:
If you add `config:environment` to the `needs` array for your unit tests, you should be able to get rid of these `beforeEach` blocks.

I believe the plan is for the isolated container that unit tests use to eventually [go away completely](https://github.com/emberjs/rfcs/pull/229), so at that point the `needs` will be unnecessary too.